HerPortal.Data/Migrations/20230628160412_AddSeparateIdColumnOnAuditDownload.cs

```diff
@@ -0,0 +1,82 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+using Npgsql.EntityFrameworkCore.PostgreSQL.Metadata;
+
+#nullable disable
+
+namespace HerPortal.Data.Migrations
+{
+    public partial class AddSeparateIdColumnOnAuditDownload : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropPrimaryKey(
+                name: "PK_AuditDownloads",
+                table: "AuditDownloads");
+
+            migrationBuilder.AlterColumn<string>(
+                name: "UserEmail",
+                table: "AuditDownloads",
+                type: "text",
+                nullable: true,
```

Comment:
I guess this is automatic because it isn't part of the primary key anymore, but it shouldn't really be nullable (same for custodian code)